core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -56,7 +56,7 @@
     private Duration minQueryExpireAge = new Duration(15, TimeUnit.MINUTES);
     private int maxQueryHistory = 100;
     private int maxQueryLength = 1_000_000;
-    private int maxStageCount = 100;
+    private int maxStageCount = 150;
```

Comment:
We do benchmark TCP-DS and it works. Why do you observe such high number of stages?